test/media/segment_index_unit.js

```diff
@@ -955,9 +955,11 @@ describe('SegmentIndex', /** @suppress {accessControls} */ () => {
    * @param {number} startTime
    * @param {number} endTime
    * @param {!Array.<!shaka.media.SegmentReference>=} partialReferences
+   * @param {!shaka.extern.HlsAes128Key} hlsAes128Key
```

Comment:
nit: If this is an optional parameter, the type should end with an equals sign to signify that.

Reply:
Thank you for your review. I fixed it!